types/superagent-retry-delay/index.d.ts

```diff
@@ -0,0 +1,72 @@
+// Type definitions for superagent-retry-delay 2.6
+// Project: https://github.com/luispabon/superagent-retry-delay#readme
+// Definitions by: Aaron Ross <https://github.com/me>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.9
+
+import { SuperAgentRequest, SuperAgentStatic, Request, Response } from "superagent";
+import * as stream from "stream";
+import * as cookiejar from "cookiejar";
+
+// these types should be exported by `superagent` but are not -------
+
+type CallbackHandler = (err: any, res: Response) => void;
+
+type Serializer = (obj: any) => string;
+
+type BrowserParser = (str: string) => any;
+
+type NodeParser = (res: Response, callback: (err: Error | null, body: any) => void) => void;
+
+type Parser = BrowserParser | NodeParser;
+
+// ------------------------------------------------------------------
+
+type EnhancedSuperAgentRequest = Omit<SuperAgentRequest, "retry"> & {
+    retry(count?: number, delay?: number): SuperAgentRequest;
```

Comment:
This line is the only meaningful change; `retry()` should now accept `number` as the optinal second argument, instead of a callback function.